### DISCUSSION TITLE
Umbraco 13 - Not generating robots.txt

### DISCUSSION THREAD

**Discussion by bronzewind** [patrickdemooij9/SeoToolkit.Umbraco#281] [Q&A]
Hi, I'm using this package in an Umbraco 13 website. Everything looks good in the backoffice, it even shows a message saying that the Robots.txt was saved, however the file is not there. Any ideas?

**Comment by patrickdemooij9** [comment]
> Hi, I'm using this package in an Umbraco 13 website. Everything looks good in the backoffice, it even shows a message saying that the Robots.txt was saved, however the file is not there. Any ideas?

Hi @bronzewind 
Sorry for this very late response. I just noticed this discussion. The Robots.txt extension doesn't actually generate a robots.txt file on disk. It saves the data in the database and serves it back through a middleware.
Let me know if you have more questions!
